Examples/polling.json.default

```diff
@@ -0,0 +1,22 @@
+{
+    "playerpoll": {
+        "type": "player",
+        "servers": "server_alias_1 server_alias_2",
+        "polling_interval": "5",
+        "polling_channel": "900210429043830814",
+        "polling_role": "<@&900500899166842911>",
+        "low_threshold": "3",
+        "medium_threshold": "6",
+        "high_threshold": "9",
+        "show_scoreboard": "True"
+    },
+    "autobalance": {
+        "type": "autobalance",
+        "servers": "tdm",
+        "polling_interval": "1",
+        "polling_channel": "900210429043830814",
+        "tk_threshold": "-10",
+        "tk_action": "kick",
+        "autobalance_min_players": "6"
```

Comment:
A few things that seem wrong with this to me

1. `servers` key should be a list if you want users to be able to enter multiple aliases (`"servers": ["alias_1", "alias_2"]`
2. all the integers should be integers instead of strings (`"polling_interval": 5`, "polling_channel": 900210429043830814` etc.)
3. boolean values should be boolean instead of strings (`"show_scoreboard": true`)

Reply:
Oh and one thing I forgot to mention, for `polling_role` (and roles in general) I would just put the ID there as integer. You can easily create the tag in code if you want but extracting the ID from the string is more tedious.